lightkurve/targetpixelfile.py

```diff
@@ -109,6 +110,61 @@ def get_prf_model(self):
         return SimpleKeplerPRF(channel=self.channel, shape=self.shape[1:],
                                column=self.column, row=self.row)
 
+
+    def get_wcs(self):
+        """Returns an astropy.wcs.WCS object with the World Coordinate System
+        solution for the target pixel file.
+
+        Returns
+        -------
+        w : astropy.wcs.WCS object
+            WCS solution
+        """
+        #Astropy.wcs.WCS has noisy warnings irrelevant for Kepler/K2
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            ok = []
+            #Find extensions with valid WCS solution
+            vals = ['CRPIX', 'CRVAL']
+            for idx, hdu in enumerate(self.hdu):
+                w = WCS(hdu.header)
+                if (np.any(np.asarray([v in k for k in hdu.header.keys() for v in vals]))):
+                    ok.append(idx)
+            #Use the first extention with a valid solution.
+            ok = ok[0]
+            w = WCS(self.hdu[ok].header)
+            return (w)
+
+    def get_radec(self):
```

Comment:
I wonder if we want to have a `cadenceno` argument and return two more easy-to-use 2D arrays (ra, dec) for a specified cadence?  In this scenario, I'd give the function a more verbose name such as `get_coordinate_matrices()`?

Reply:
I like `get_coordinate_matrices()`. I can have that be passed a cadenceno, and if it's none it returns the whole lot?